sql/core/src/test/scala/org/apache/spark/sql/IntegratedUDFTestUtils.scala

```diff
@@ -197,7 +197,7 @@ object IntegratedUDFTestUtils extends SQLHelper {
 
   lazy val pythonExec: String = {
     val pythonExec = sys.env.getOrElse(
-      "PYSPARK_DRIVER_PYTHON", sys.env.getOrElse("PYSPARK_PYTHON", "python3.6"))
+      "PYSPARK_DRIVER_PYTHON", sys.env.getOrElse("PYSPARK_PYTHON", "python3"))
```

Comment:
Since this is `GitHub Action` specific issue, shall we use `export PYSPARK_PYTHON` in GitHub Action side?